lms/static/sass/discussion/views/_thread.scss

```diff
@@ -1,23 +1,37 @@
 // discussion - thread layout
 // ====================
+// NOTE: thread = (post + (responses and comments))
 
-// general thread layout
+// Table of Contents 
+// * +general thread layout
+// * +thread - wrapper styling
+// * +thread - elements - shared styles
+// * +post - individual element styling
+// * +post - answered question - collapsed comment area
+
+// +general thread layout
 body.discussion, .discussion-module {
 
   // post layout
   .discussion-post {
-    padding: ($baseline*2) ($baseline*2) $baseline ($baseline*2);
-    border-radius: 3px 3px 0 0;
+    @include padding(($baseline*1.5), ($baseline*1.5), $baseline, ($baseline*1.5));
+    @include border-radius(3px, 3px, 0, 0);
     background-color: $white;
 
+    .wrapper-post-header {
+      padding-bottom: 0;
+      margin-bottom: ($baseline*0.75);
+    }
+
     .post-header-content {
       display: inline-block;
-      width: flex-grid(9,12);
+      // width: calc(flex-grid(9,12) - $post-image-dimension);
+      width: flex-grid(8,12); 
     }
 
     .post-header-actions {
-      display: inline-block;
       @include float(right);
+      display: inline-block;
```

Comment:
ah yeah, just reordered properties here blindly without noticing that. cleaned up now. :) good catch.